src/renderer/src/hooks/useAfterLogin.ts

```diff
@@ -291,6 +297,57 @@ const useMigrationNeo3Notification = () => {
   }, [balanceQuery.data, balanceQuery.isLoading, dispatch, t, unreadNotificationsRef])
 }
 
+const useVotingNeo3Notification = () => {
+  const { ownAccounts } = useOwnAccountsSelector()
+  const { unreadNotificationsRef } = useUnreadNotificationsSelector()
+  const dispatch = useAppDispatch()
+  const { t } = useTranslation('hooks', { keyPrefix: 'useVotingNeo3Notification' })
+
+  const voteNeo3Accounts = useMemo(() => ownAccounts.filter(account => account.blockchain === 'neo3'), [ownAccounts])
+
+  const voteDetailsQuery = useVoteNeo3GetVoteDetailsByAddresses(voteNeo3Accounts)
+
+  const alreadyNotifiedRef = useRef(false)
+
+  useEffect(() => {
+    if (voteDetailsQuery.isLoading || alreadyNotifiedRef.current) return
+
+    alreadyNotifiedRef.current = true
+
+    voteDetailsQuery.data.forEach(voteDetails => {
+      if (voteDetails.candidatePubKey || voteDetails.neoBalance <= 0) return
```

Comment:
In [another PR](https://github.com/CityOfZion/neon-wallet-mobile/pull/2157#discussion_r2163681065) you removed it.

Reply:
It is wrong there